halibot/halibot.py

```diff
@@ -64,6 +67,22 @@ def shutdown(self):
 
 		self.log.info("Halibot shutdown. Threads left: " + str(threading.active_count()))
 
+	def _check_version(self, obj):
+		v = Version(self.VERSION)
+		if not hasattr(obj, "HAL_MINIMUM"):
+			self.log.warn("Module class '{}' does not define a minimum version, trying to load anyway...".format(obj.__class__.__name__))
+			return True
```

Comment:
Should we still check if there is a `HAL_MAXIMUM` even if there is not a `HAL_MINIMUM`?

Reply:
Good point, probably not. Will fix shortly.